demo/appointment_noshow_tutorial.ipynb

```diff
@@ -0,0 +1,366 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cd.list_problems()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case, we will define the problem as ‘Missed Appointment’ to predict whether a patient will miss their next appointment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# select problem\n",
+    "cutoff = cd.select_problem('MissedAppointmentProblemDefinition')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## AutoML\n",
+    "Automated machine learning composes from two main phases:\n",
+    "1. automated feature engineering: through autofe, we extract information called features. Finding the features is crucial for building data models and help in finding a satisfactory answer and interpreting the dataset as a whole.\n",
+    "2. automated modeling: in automated modeling, the library supports running multiple machine learning algorithms and optimizes its hyperparamters in order to find the most optimal model.\n",
+    "\n",
+    "Typically, this phase is complex and comprises of many elements, but **Cardea** provides an easier way of handling both phases."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Built 13 features\n",
+      "Elapsed: 00:41 | Remaining: 00:00 | Progress: 100%|██████████| Calculated: 10/10 chunks\n"
+     ]
+    }
+   ],
+   "source": [
+    "# feature engineering\n",
+    "feature_matrix = cd.generate_features(cutoff[:1000]) # first 1000 records"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# shuffle the dataframe\n",
+    "feature_matrix = feature_matrix.sample(frac=1)\n",
+    "\n",
+    "# pop the target labels\n",
+    "y = list(feature_matrix.pop('label'))\n",
+    "X = feature_matrix.values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The pipeline variable represents the order in which machine learning algorithms are executed. It can be used to compare models together by specifying multiple algorithms in different lists. Such as:\n",
+    "```\n",
+    "pipeline = [['sklearn.ensemble.RandomForestClassifier'], \n",
+    "            ['sklearn.neighbors.KNeighborsClassifier']]\n",
+    "```\n",
+    "Here we execute two different models, the first one being Random Forest and the second is Logistic Regression.\n",
+    "In addition, you can use the pipeline to create your own encoding and modeling pipeline where the data crosses several algorithms to create the prediction model. Such as:\n",
+    "```\n",
+    "pipeline = [['sklearn.ensemble.RandomForestClassifier', 'sklearn.naive_bayes.MultinomialNB'], \n",
```

Comment:
we might need a `\n` here to avoid this:

![image](https://user-images.githubusercontent.com/4410771/56287833-62a3e500-60eb-11e9-9f0c-772b5e71425b.png)

Also, I am not sure if there is a typo, you wanted to use:

```
[
    ['sklearn.ensemble.RandomForestClassifier',
     'sklearn.naive_bayes.MultinomialNB'], 
    ['sklearn.neighbors.KNeighborsClassifier']
]
```

or

```
[
    ['sklearn.ensemble.RandomForestClassifier'],
    ['sklearn.naive_bayes.MultinomialNB'], 
    ['sklearn.neighbors.KNeighborsClassifier']
]
```
?

Reply:
I meant to use the first one, but maybe this example isn't the clearest. I updated the notebook to show how you can use the pipeline to do preprocessing prior to modeling.
```
[
    ['sklearn.preprocessing.StandardScaler', 'sklearn.ensemble.RandomForestClassifier'], 
    ['sklearn.neighbors.KNeighborsClassifier']
]
```

Reply:
Understood. I was just asking because you were using the first one in the description, but then in the next line (python code `In [10]`) you were actually using the second one.